pkg/registry/admissionreview/admission_review.go

```diff
@@ -30,6 +30,10 @@ func (r *REST) New() runtime.Object {
 	return &admissionv1beta1.AdmissionReview{}
 }
 
+func (r *REST) Destroy() {
```

Comment:
Took me a minute to figure out why we needed to add `Destroy()` given it's a method on (Standard)Storage but L19-21 aren't declaring that we're implementing either of those interfaces.

It's [this chunk](https://github.com/openshift/hive/blob/40fcaecc31022b16fcf8b5669a7344240c7065cc/vendor/github.com/openshift/generic-admission-server/pkg/apiserver/apiserver.go#L150-L155) that's using the REST object as a Storage.

So IMO it would be a good look to add
```go
var _ rest.Storage = &REST{}
```
above with the others.